epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.kt

```diff
@@ -998,7 +1089,7 @@ internal class GeneratedModelWriter(
             if (attr is ViewAttributeInfo && attr.generateStringOverloads) {
                 methods.addAll(StringOverloadWriter(modelInfo, attr, configManager).buildMethods())
             } else {
-                if (attr.isViewClickListener) {
+                if (attr.isViewClickListener || attr.isViewLongClickListener) {
```

Comment:
^ this is a change (though not a functional one, I modified `isViewClickListener` so I could use it elsewhere).